page/join_group.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  page: page
+}
+
+function page (req, res) {
+  console.log('making join group page...')
```

Comment:
unnecessary

Reply:
will be removed

Reply:
Why did you mark this conversation as resolved?